# Generated by Django 4.1.1 on 2022-09-28 21:47

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('CompanyMachine', '0002_auto_20220923_1232'),
    ]

    operations = [
        migrations.CreateModel(
            name='User_Medidores_Model',
            fields=[
                ('id', models.BigAutoField(primary_key=True, serialize=False)),
                ('context', models.TextField()),
                ('registerd_at', models.DateTimeField(blank=True, editable=False)),
                ('model', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='CompanyMachine.medidoresmodel')),
                ('model_user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.CreateModel(
            name='User_Calibraciones_Model',
            fields=[
                ('id', models.BigAutoField(primary_key=True, serialize=False)),
                ('context', models.TextField()),
                ('registerd_at', models.DateTimeField(blank=True, editable=False)),
                ('model', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='CompanyMachine.calibracionesmodel')),
                ('model_user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
        migrations.AddField(
            model_name='calibracionesmodel',
            name='user',
            field=models.ManyToManyField(through='CompanyMachine.User_Calibraciones_Model', to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='medidoresmodel',
            name='user',
            field=models.ManyToManyField(through='CompanyMachine.User_Medidores_Model', to=settings.AUTH_USER_MODEL),
        ),
    ]
